src/ConfigManager.ts

```diff
@@ -0,0 +1,38 @@
+import { readFileSync, writeFileSync } from "fs";
+
+type Config = {
+  winEloGain: number;
+  loseEloLoss: number;
+  mvpBonus: number;
+  roles: {
+    blueTeamRole: string;
+    redTeamRole: string;
+    captainRole: string;
+    organiserRole: string;
+  };
+  channels: {
+    registration: string;
+    mapVote: string;
+    minerushingVote: string;
```

Comment:
These won't have their own channels, there will be an announcements channel and there'll be 
Message 1: Announcement + info
Message 2: if map vote true: map vote message + emojis to react below
Message 3: if minerushing vote true: minerushing message + emojis to react below
```suggestion
```

Reply:
alright, i'll fix that soon